src/test/help.js

```diff
@@ -0,0 +1,40 @@
+/*eslint no-invalid-this: "off" */
+import { exec } from 'child_process'
+
+const usage = `
+  Usage: atomiq [options] [command]
```

Comment:
It's probably good enough to just verify that the command responds with the main usage first line. I don't think we need to validate further since help is auto-generated and therefore any exposed command will be listed. This will just end up being something brittle that will have to be updated every time we add commands or update their help descriptions.